### ISSUE TITLE
Codehilite extension breaks if code contains unicode

### ISSUE THREAD

**Issue by last-partizan** [django-wiki/django-wiki#607]
If code block contains unicode, then extension fails with next exception

![s1](https://cloud.githubusercontent.com/assets/301015/24200552/1dfa28fc-0f16-11e7-9f62-192477131cf6.png)

This is fixed by simply using unicode instead of string here

https://github.com/django-wiki/django-wiki/blob/master/wiki/core/markdown/mdx/codehilite.py#L28

```
-                html = """<div class="codehilite-wrap">{}</div>""".format(html)
+                html = u"""<div class="codehilite-wrap">{}</div>""".format(html)
```

or by adding `from __future__ import unicode_literals`

- Python 2.7.13
- Django 1.10
- wiki 0.2.1

**Comment by benjaoming** [comment]
Thanks for reporting this!

> or by adding from __future__ import unicode_literals

This is the correct fix, as we have to be Py2+3.

**Comment by benjaoming** [comment]
Fixed in #617, will be released in 0.2.3
